test/test_file_processing.py

```diff
@@ -254,7 +254,7 @@ def test_reprocess_recording(self, helper, file_processing):
 
         # Now finalise processing.
         file_processing.put(recording, success=True, complete=True, new_object_key="some_key")
-        check_recording(admin, recording, processingState="FINISHED", fileKey="some_key")
+        check_recording(admin, recording, processingState="FINISHED")
```

Comment:
Ahh... the file processing API is an internal API which needs to include fileKey and rawFileKey. if `get()` in recordingUtil.js is used for both places then the field removal needs to happen at a higher layer.

Sorry that I missed this detail.

Reply:
Actually, never mind. I should have checked the code first. I see the file processing API gets the recording in a different way which still includes fileKey and rawFileKey.

Is there some other way that this test can confirm that the correct object key was set in the DB? The only approach I can think of is to actually upload some data to the object store at some key, set the recording to use that key and then download the recording to ensure the expected data is returned. 

Or, have a peek directly in the DB.

Perhaps not worth it...

Reply:
See update, I've used the where param to include it as part of the query, which is the only way I can think of (although if you should be able to query based on fileKey is another issue for #180)